### ISSUE TITLE
What about patina?

### ISSUE THREAD

**Issue by ThEnchanted** [kennytv/list-of-shame#74]
Hello!
Could I ask a question about server core called "patina"? Didn't have too many opinions mostly from Russian (Mostly positive) but still not enough information and wanna know about it more...

**Comment by kennytv** [comment]
Patina is a work with plainly useless patches, as well as patches from Yatopia, and is barely maintained anymore anyways, so I would recommend not using it.

**Comment by ThEnchanted** [comment]
Could you add it to the list of shame? please?

**Comment by cubelius** [comment]
I agree. I do not recommend using Patina as a server fork. If you do the tests, this core loses in performance with many players. The patches are really useless (but not as much as in Yatopia).

Kenny, what do you think about Pufferfish? Fork came out recently, but haven't checked it out.

**Comment by Dreeam-qwq** [comment]
> I agree. I do not recommend using Patina as a server fork. If you do the tests, this core loses in performance with many players. The patches are really useless (but not as much as in Yatopia).
> 
> Kenny, what do you think about Pufferfish? Fork came out recently, but haven't checked it out.

Pufferfish is a good fork which brings airplane features in 1.18.1

**Comment by Outfit8TSB** [comment]
Pufferfish never updated according to @granny.
> That's flawed from the start. Tuinity was merged into Paper, which means it's dead. Pufferfish is a continuation of Airplane, which means Airplane is obsolete. That leaves just Pufferfish (which still hasn't updated to 1.20.6)

Besides, This is a issue About Patina, Not Pufferfish. please refrain from talking about pufferfish.
